micrometer-observation/src/main/java/io/micrometer/observation/SimpleObservation.java

```diff
@@ -366,29 +356,22 @@ public void reset() {
          */
         @Override
         public void makeCurrent() {
+            Deque<SimpleObservation> observations = new ArrayDeque<>();
```

Comment:
After our fixes in `ObservationThreadLocalAccessor` calling this method shouldn't be needed anymore. Do you have any use case for it?

Reply:
No, after the gRPC fix, I don't have a particular usage for `makeCurrent` method "anymore".